File: app/src/main/java/ru/melod1n/vk/concurrent/LowThread.java
package ru.melod1n.vk.concurrent;

import android.os.Process;

public class LowThread extends Thread {

    public LowThread(Runnable runnable) {
        super(runnable);
    }

    @Override
    public void run() {
        Process.setThreadPriority(Process.THREAD_PRIORITY_BACKGROUND);
        super.run();
    }
}

File: app/src/main/java/ru/melod1n/vk/concurrent/LowThread.kt
package ru.melod1n.vk.concurrent

import android.os.Process

class LowThread(runnable: Runnable) : Thread(runnable) {
    override fun run() {
        Process.setThreadPriority(Process.THREAD_PRIORITY_BACKGROUND)
        super.run()
    }

    constructor(runnable: () -> Unit) : this(Runnable(runnable))
}